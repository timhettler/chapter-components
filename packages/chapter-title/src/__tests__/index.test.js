import React from 'react';
import Block from '../';
import renderer from 'react-test-renderer';

it(`${Block.name} renders correctly`, () => {
  const tree = renderer.create(<Block title="FOO" subtitle="BAR" />).toJSON();
  expect(tree).toMatchSnapshot();
});

it(`${Block.name} renders correctly without title`, () => {
  const tree = renderer.create(<Block subtitle="BAR" />).toJSON();
  expect(tree).toMatchSnapshot();
});
