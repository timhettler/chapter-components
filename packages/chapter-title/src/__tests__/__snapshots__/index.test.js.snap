// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChapterTitle renders correctly 1`] = `
<header
  className="header is-dark is-default"
>
  <h1
    className="title"
  >
    FOO
  </h1>
  <h2
    className="subtitle"
  >
    <div
      className="line"
    >
      <span
        className="letter"
        style={
          Object {
            "transform": null,
            "transition": "opacity 100ms 40ms, transform 100ms 40ms",
          }
        }
      >
        B
      </span>
      <span
        className="letter"
        style={
          Object {
            "transform": null,
            "transition": "opacity 200ms 80ms, transform 200ms 80ms",
          }
        }
      >
        A
      </span>
      <span
        className="letter"
        style={
          Object {
            "transform": null,
            "transition": "opacity 300ms 120ms, transform 300ms 120ms",
          }
        }
      >
        R
      </span>
    </div>
  </h2>
</header>
`;
